### PR TITLE
docs/backends/cos: add necessary permission list for using the backend

### PR DESCRIPTION
**Pull request by davendu**
Give a list of permissions required for the COS state backend. This can be a reference if users want to define stricter policies.

## Target Release

1.4.x or next available document release

### CODE REVIEW

**Comment by crw** [comment]
@likexian, can you please review and leave a comment to approve as appropriate?

I will also notify the docs team for a docs review. Thanks for the submission!

**Comment by likexian** [comment]
LGTM

**Comment by davendu** [comment]
Hi guys, any future suggestions?

File: website/docs/language/settings/backends/cos.mdx [hunk 1]
```diff
@@ -9,7 +9,10 @@ description: >-
 
 Stores the state as an object in a configurable prefix in a given bucket on [Tencent Cloud Object Storage](https://intl.cloud.tencent.com/product/cos) (COS).
 
-This backend supports [state locking](/terraform/language/state/locking).
+This backend supports [state locking](/terraform/language/state/locking). You need following permissions for your credentials:
+
+* `CreateTag`, `DeleteTag` and `DescribeTags` on the tag key `tencentcloud-terraform-lock`.
```
Comment (rkoron007) [inline website/docs/language/settings/backends/cos.mdx:14]: Small suggestion for using `-` for markdown lists, removing the final period on the sentence fragment, and adding an oxford comma:
```suggestion
- `CreateTag`, `DeleteTag`, and `DescribeTags` on the tag key `tencentcloud-terraform-lock`
```

**Review by rkoron007 — COMMENTED** [review]
Hello @davendu! 👋  I apologize for the delay here; thank you so much for this PR! I left a few small suggestions for style, but _please_ feel free to push back on anything that doesn't feel good to you.

**Comment by davendu** [comment]
Thanks @rkoron007, I think your suggestions are great! Have accepted and pushed a new commit.

**Review by rkoron007 — APPROVED** [review]
Looks great to me; thank you so much for contributing! ✨

**Pull request merged**